pkg/cmd/task/list.go

```diff
@@ -73,12 +73,13 @@ func printTaskDetails(out io.Writer, p cli.Params) error {
 
 	tasks, err := listTaskDetails(cs.Tekton, p.Namespace())
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to list tasks from %s namespace\n", p.Namespace())
-		return err
+		fmt.Fprintf(out, emptyMsg)
```

Comment:
same; must fail if the object does not exist

Reply:
**tkn task list** does not take a particular name, so we could only print the message like other list commands.